modules/ztp-configuring-the-cluster-for-a-disconnected-environment.adoc

```diff
@@ -28,23 +28,32 @@ If you enable TLS for the HTTP server, you must confirm the root certificate is
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: assisted-installer-mirror-config
-  namespace: assisted-installer
+  name: assisted-installer-config-map
+  namespace: "<infrastructure-operator-namespace>" <1>
   labels:
     app: assisted-service
 data:
-  ca-bundle.crt: <certificate> <1>
-  registries.conf: |  <2>
+  ca-bundle.crt: | <2>
+    -----BEGIN CERTIFICATE-----
+    certificate contents
+    -----END CERTIFICATE-----
+
+  registries.conf: | <3>
     unqualified-search-registries = ["registry.access.redhat.com", "docker.io"]
 
     [[registry]]
-      location = <mirror_registry_url>  <3>
-      insecure = false
-      mirror-by-digest-only = true
+       prefix = ""
+       location = "quay.io/example-repo" <4>
+       mirror-by-digest-only = true
+
+       [[registry.mirror]]
+       location = "mirror1.registry.corp.com:5000/example-repo" <5>
 ----
-<1> The mirror registry’s certificate used when creating the mirror registry.
-<2> The configuration file for the mirror registry. The mirror registry configuration adds mirror information to `/etc/containers/registries.conf` in the Discovery image. The mirror information is stored in the `imageContentSources` section of the `install-config.yaml` file when passed to the installation program. The Assisted Service pod running on the HUB cluster fetches the container images from the configured mirror registry.  
-<3> The URL of the mirror registry.
+<1> The `ConfigMap` namespace must be the same as the namespace of the Infrastructure Operator.
+<2> The mirror registry’s certificate used when creating the mirror registry.
+<3> The configuration file for the mirror registry. The mirror registry configuration adds mirror information to `/etc/containers/registries.conf` in the Discovery image. The mirror information is stored in the `imageContentSources` section of the `install-config.yaml` file when passed to the installation program. The Assisted Service pod running on the hub cluster fetches the container images from the configured mirror registry.
```

Comment:
```suggestion
<3> The configuration file for the mirror registry. The mirror registry configuration adds mirror information to the `/etc/containers/registries.conf` directory in the Discovery image. The mirror information is stored in the `imageContentSources` section of the `install-config.yaml` file when the information is passed to the installation program. The Assisted Service pod that runs on the hub cluster fetches the container images from the configured mirror registry.
```
- Is `/etc/containers/registries.conf` a file, a directory, or something else?
- Please check my revision for accuracy. The original is a little unclear about what is being passed to the installation program. I assumed it was the mirror information.